sdks/java/io/kafka/pom.xml

```diff
@@ -29,6 +29,10 @@
   <name>Apache Beam :: SDKs :: Java :: IO :: Kafka</name>
   <description>Library to read Kafka topics.</description>
 
+  <properties>
+    <kafka.clients.version>0.10.1.1</kafka.clients.version>
```

Comment:
Can we keep the default to 0.9.0.1? Looks like in Kafka, older clients can connect to newer Kafka servers, but not the other way around.